util/trace/trace.go

```diff
@@ -8,22 +8,29 @@ import (
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc"
 	"go.opentelemetry.io/otel/propagation"
-	"go.opentelemetry.io/otel/sdk/resource"
+	sdkresource "go.opentelemetry.io/otel/sdk/resource"
```

Comment:
It is better to use resource here

Reply:
Why? The other SDK imports are aliased as well and it makes for a nice difference of local variables vs imports